src/components/Dashboard.tsx

```diff
@@ -0,0 +1,160 @@
+import React, { useEffect } from 'react'
```

Comment:
`useEffect` not in use

Reply:
275040f18ccef37f150fb988502d35fb95aeff78